### DISCUSSION TITLE
11-25

### DISCUSSION THREAD

**Discussion by oauch** [ChristmasTest/christmas#3] [Daily_Meeting]
### 🎅🏻 회의 내용

- docker(?)
- jest
- sentry
- story book
- 디자이너를 구하기 위한 홍보글 작성

### ❄️ 오늘 할일

- 홍보완료(?)
- storybook 세팅 [윤혁님]
- jest 세팅 [수님]
- ga, hotjar [슬희님]
- 카카오톡 공유하기 & meta 태그 (seo) 완성시키기 [수영님]

### ⛄️ TMI

다들 옷 두껍게 입으세요 ☡

**Comment by hanseulhee** [comment]
10팀 끝 ~! 🥺🎅🏻
